solidity/contracts/converter/types/liquidity-pool-v2/LiquidityPoolV2Converter.sol

```diff
@@ -31,43 +30,22 @@ contract LiquidityPoolV2Converter is LiquidityPoolConverter {
     mapping (IERC20Token => ISmartToken) private reservesToPoolTokens;  // maps each reserve to its pool token
     mapping (ISmartToken => IERC20Token) private poolTokensToReserves;  // maps each pool token to its reserve
 
-    uint8 public amplificationFactor = 20;  // factor to use for conversion calculations (reduces slippage)
-    uint256 public externalRatePropagationTime = 1 hours;  // the time it takes for the external rate to fully take effect
-    uint256 public prevConversionTime;  // previous conversion time in seconds
-
-    // factors used in fee calculations
-    uint32 public lowFeeFactor = 200000;
-    uint32 public highFeeFactor = 800000;
+    Fraction public externalRate;           // conversion rate of 1 primary token in secondary tokens

```

Comment:
that's actually the external rate, not a conversion rate

Reply:
Fixed.